src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/Slider/SliderHandler.Android.cs

```diff
@@ -0,0 +1,124 @@
+	{
+		static ColorStateList? DefaultProgressTintList { get; set; }
+		static ColorStateList? DefaultProgressBackgroundTintList { get; set; }
+		static PorterDuff.Mode? DefaultProgressTintMode { get; set; }
+		static PorterDuff.Mode? DefaultProgressBackgroundTintMode { get; set; }
+		static ColorFilter? DefaultThumbColorFilter { get; set; }
+
+		SeekBarChangeListener? _seekBarChangeListener;
+
+		protected override SeekBar CreateNativeView()
+		{
+			var slider = new SeekBar(Context);
+			_seekBarChangeListener = new SeekBarChangeListener();
+			slider.SetOnSeekBarChangeListener(_seekBarChangeListener);
+			return slider;
+		}
+
+
+		public override void TearDownNativeView(SeekBar nativeView)
+		{
+			nativeView.SetOnSeekBarChangeListener(null);
+		}
+
+		protected override void SetUpNewVirtualView(ISlider newView)
+		{
+			if(_seekBarChangeListener != null)
+				_seekBarChangeListener.VirtualView = newView;
+		}
+
+		protected override void TearDownOldVirtualView(ISlider oldView)
+		{
+			if (_seekBarChangeListener != null)
+				_seekBarChangeListener.VirtualView = null;
+		}
+
+		protected override void SetupDefaults(SeekBar nativeView)
+		{
+			DefaultThumbColorFilter = nativeView.Thumb?.GetColorFilter();
+			DefaultProgressTintMode = nativeView.ProgressTintMode;
+			DefaultProgressBackgroundTintMode = nativeView.ProgressBackgroundTintMode;
+			DefaultProgressTintList = nativeView.ProgressTintList;
+			DefaultProgressBackgroundTintList = nativeView.ProgressBackgroundTintList;
+		}
+
+		public static void MapMinimum(SliderHandler handler, ISlider slider)
+		{
+			ViewHandler.CheckParameters(handler, slider);
+
+			handler.TypedNativeView?.UpdateMinimum(slider);
+		}
+
+		public static void MapMaximum(SliderHandler handler, ISlider slider)
+		{
+			ViewHandler.CheckParameters(handler, slider);
+
+			handler.TypedNativeView?.UpdateMaximum(slider);
+		}
+
+		public static void MapValue(SliderHandler handler, ISlider slider)
+		{
+			ViewHandler.CheckParameters(handler, slider);
+
+			handler.TypedNativeView?.UpdateValue(slider);
+		}
+
+		public static void MapMinimumTrackColor(SliderHandler handler, ISlider slider)
+		{
+			ViewHandler.CheckParameters(handler, slider);
+
+			handler.TypedNativeView?.UpdateMinimumTrackColor(slider, DefaultProgressBackgroundTintList, DefaultProgressBackgroundTintMode);
+		}
+
+		public static void MapMaximumTrackColor(SliderHandler handler, ISlider slider)
+		{
+			ViewHandler.CheckParameters(handler, slider);
+
+			handler.TypedNativeView?.UpdateMaximumTrackColor(slider, DefaultProgressTintList, DefaultProgressTintMode);
+		}
+
+		public static void MapThumbColor(SliderHandler handler, ISlider slider)
+		{
+			ViewHandler.CheckParameters(handler, slider);
+
+			handler.TypedNativeView?.UpdateThumbColor(slider, DefaultThumbColorFilter);
+		}
+
+		internal class SeekBarChangeListener : Java.Lang.Object, SeekBar.IOnSeekBarChangeListener
+		{
+			public ISlider? VirtualView { get; set; }
```

Comment:
I changed this to be settable because the handler might get its virtual view swapped out. We can't just instantiate this with a VirtualView. The other option here would be to instantiate this with a Func<ISlider> but I think this is fine. This way we don't have to resolve a "Func" each time we want the view

Reply:
Comet re-uses Handlers. So that makes sense.